README.md

```diff
@@ -44,14 +44,14 @@
 
     ``` yaml    
     imports:
-        - { resource: "@SynoliaSyliusMaintenancePlugin/Resources/config/config.yaml" }
+        - { resource: "@SynoliaSyliusMaintenancePlugin/config/config.yaml" }
```

Comment:
Can be considered as BC break. Maybe we should keep that for the v2 compatible and keep the old style for the v1 ? 

Or at least it should be indicated in a UPGRADE.md file ;)